src/run-es.js

```diff
@@ -0,0 +1,72 @@
+const fs = require('fs');
+const logger = require('../log.js');
+const nconf = require('nconf');
```

Comment:
it is usually makes your code a bit more readable, if you order your requires in the following way:

1. built-in modules,
2. modules from npm
3. your modules